src/pages/Whattodo.js

```diff
@@ -21,6 +21,11 @@ export default function Whattodo() {
       console.error('Error fetching data:', error);
     }
   };
+  
+  function handleDelete(activity) {
+    const newlist = Task.filter((list) => list.activity !== activity.activity);
```

Comment:
nit: this doesn't handle duplicate activities, maybe we have to use the index or unique id to indicate which one the user is trying to delete. 

Also add tests to confirm behavior  
consider the following test cases 

1. Delete duplicate items on the list
2. Delete first and last item on the list